src/test/ui/sanitize/memory.rs

```diff
@@ -1,7 +1,7 @@
 // needs-sanitizer-support
 // needs-sanitizer-memory
 //
-// compile-flags: -Z sanitizer=memory -Zsanitizer-memory-track-origins -O
+// compile-flags: -Z sanitizer=memory -Zno-validity-invariant-checks -Zsanitizer-memory-track-origins -O
```

Comment:
Before landing – the flag should be inverted and enabling it by default would need to be considered alongside stabilization of this feature as a whole.